apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ProjectMetricsResourceTest.java

```diff
@@ -556,23 +568,41 @@ private Map<String, Long> createSpans(
                     .map(trace -> factory.manufacturePojo(Span.class).toBuilder()
                             .projectName(projectName)
                             .traceId(trace.id())
-                            .usage(usageNames.stream()
-                                    .collect(Collectors.toMap(name -> name, n -> RANDOM.nextInt())))
+                            .usage(usageNames == null
+                                    ? null
+                                    : usageNames.stream().collect(
+                                            Collectors.toMap(name -> name, n -> RANDOM.nextInt())))
                             .build())
                     .toList();
 
             spanResourceClient.batchCreateSpans(spans, API_KEY, WORKSPACE_NAME);
 
             return spans.stream().map(Span::usage)
+                    .filter(Objects::nonNull)
                     .flatMap(i -> i.entrySet().stream())
                     .collect(Collectors.groupingBy(Map.Entry::getKey))
                     .entrySet().stream()
                     .collect(Collectors.toMap(
                             Map.Entry::getKey,
                             entry -> entry.getValue().stream()
-                                    .filter(usage -> usage.getKey().equals(entry.getKey()))
+                                    .filter(u -> u.getKey().equals(entry.getKey()))
```

Comment:
Better flavor readability.
```suggestion
                                    .filter(usage -> usage.getKey().equals(entry.getKey()))
```

Reply:
This change was committed by accident. Thanks!